### DISCUSSION TITLE
TE-TM superposition

### DISCUSSION THREAD

**Discussion by OwlyWizard** [kc-ml2/meent#80] [Q&A]
Hi, 
I was wondering if it was possible to run a simulation with a polarization other than 0 or 90° (like 45 or 135) ? Of course I could do TE and TM simulations separatly and then recombine the result afterward but I was thinking of a simpler way to do it.

Thanks!

**Comment by yonghakim** [comment]
Hi,

https://github.com/kc-ml2/meent/blob/main/tutorials/01-modeling-and-emsolver.ipynb

In this notebook, see cell # 10. You can play with `pol`. 0.5 is the 45 degree. 
To be precise, `pol` is a wrapper of `psi` which is defined as the angle between E field and the plane of incidence (see Moharam or our paper for figure). This is for beginner who doesn't know about the polarization.
You can directly control `psi` without any problems.

**Comment by OwlyWizard** [comment]
Thanks!
